tasks/http_apache.yml

```diff
@@ -13,15 +13,16 @@
     state: present
     regexp: "{{ item.regexp }}"
     line: "{{ item.line }}"
-    backrefs: yes
+    backrefs: true
   with_items:
-    - { regexp: 'opcache.enable=0', line: 'opcache.enable=1' }
-    - { regexp: 'opcache.enable_cli', line: 'opcache.enable_cli=1' }
-    - { regexp: 'opcache.interned_strings_buffer', line: 'opcache.interned_strings_buffer=8' }
-    - { regexp: 'opcache.max_accelerated_files', line: 'opcache.max_accelerated_files=10000' }
-    - { regexp: 'opcache.memory_consumption', line: 'opcache.memory_consumption=128' }
-    - { regexp: 'opcache.save_comments', line: 'opcache.save_comments=1' }
-    - { regexp: 'opcache.revalidate_freq', line: 'opcache.revalidate_freq=1' }
+    - {regexp: 'opcache.enable=0', line: 'opcache.enable=1'}
```

Comment:
Is ansible-lint suggested removing trailing spaces after { and before } ?

Reply:
Exactly, and apparently it increases the score in ansible galaxy.